src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -2202,3 +2314,117 @@ def _validate_aci_location(norm_location):
     if norm_location not in aci_locations:
         raise CLIError('Azure Container Instance is not available at location "{}".'.format(norm_location) +
                        ' The available locations are "{}"'.format(','.join(aci_locations)))
+
+
+def osa_list(cmd, client, resource_group_name=None):
+    if resource_group_name:
+        managed_clusters = client.list_by_resource_group(resource_group_name)
+    else:
+        managed_clusters = client.list()
+    return _remove_osa_nulls(list(managed_clusters))
+
+
+def openshift_create(cmd, client, resource_group_name, name,  # pylint: disable=too-many-locals
+                     location=None,
+                     compute_vm_size="Standard_D4s_v3",
+                     compute_count=3,
+                     fqdn='',
```

Comment:
Remove this empty string and place it before location so the CLI will automatically recognize that it is required.